wasm/main.go

```diff
@@ -0,0 +1,61 @@
+// To compile use
+// GOOS=js GOARCH=wasm go build -o main.wasm main.go
+//
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"syscall/js"
+
+	"github.com/drand/drand/chain"
+	"github.com/drand/drand/key"
+)
+
+var done = make(chan struct{})
+
+func main() {
+	callback := js.FuncOf(verifyBeacon)
```

Comment:
Is this interface going to grow, or do we just need verifyBeacon?

if there's more, we should have an interface of what methods we're exposing to JS, and use a level of abstraction.

Reply:
Right now the scope is only for the website.
Also, the WASM is only there for the cryptographic operations, so all the rest (fetching a beacon, chain info etc) can be done by pure JS.